Dockerfile

```diff
@@ -81,6 +81,12 @@ RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" \
   && php composer-setup.php --install-dir=/bin --filename=composer --version=1.10.16 \
   && php -r "unlink('composer-setup.php');"
 
+# Install the global drush launcher
+RUN curl -OL https://github.com/drush-ops/drush-launcher/releases/latest/download/drush.phar \
```

Comment:
Could we pin this to a specific version rather than `latest`? Pinning versions increases our maintenance overhead, but means rebuilding an image should always yield the same results, even if a newer version of drush is released. We saw a problem not so long ago when Composer had a major update with a breaking change that broke our build. That led us to pin Composer up on L81